src/wido-compound-sdk.ts

```
@@ -188,25 +191,18 @@ export class WidoCompoundSdk {
       throw new Error("There is no loan provider to enable this swap");
     }
 
-    // fetch Wido Token Manager address
-    const tokenManager = await getWidoSpender({
-      chainId: chainId,
-      fromToken: fromAsset.address,
-      toChainId: chainId,
-      toToken: toAsset.address
-    })
+    const tokenManager = widoTokenManager[chainId]
 
     // check values and set defaults
-    const supported = quoteResponse.isSupported;
-    const toAmount = supported && quoteResponse.toTokenAmount
+    const toAmount = quoteResponse.toTokenAmount
       ? quoteResponse.toTokenAmount
       : "0";
-    const minToAmount = supported && quoteResponse.minToTokenAmount
-      ? quoteResponse.minToTokenAmount
+    const minToAmount = minToTokenAmount
```

Let's raise an error when the minToTokenAmount is not present instead of setting it to 0